integration-cli/docker_cli_help_test.go

```diff
@@ -100,5 +87,9 @@ func TestCmdHelpWidth(t *testing.T) {
 		}
 	}
 
+	if count < 10 {
```

Comment:
hmmm I guess forcing people to update this each time a new cmd is added isn't so bad....  at least they can't claim they didn't know because the test will fail.  :-)  ok will "make it so".